src/Examples/Flowsheets/power_generation/rsofc/rsofc_soec_flowsheet.py

```diff
@@ -2334,7 +2334,7 @@ def average_current_density_constraint(b, t):
         "ma57_pivtolmax": 0.1,
         "OF_ma57_automatic_scaling": "yes",
     }
-    solver.solve(m.soec_fs, tee=True, options=options, symbolic_solver_labels=True)
+    solver.solve(m, tee=True, options=options, symbolic_solver_labels=True)
```

Comment:
Do we need symbolic solver labels here?

Reply:
Probably not, but it was there before so I didn't mess with it.  This part isn't my model, I just was improving code reuse, and getting it to a state where I could test some things.  @bpaul4 can take a look in another PR or I'll come back if I can get time to take a closer look.